barcodes_generator_location/__manifest__.py

```diff
@@ -2,13 +2,13 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Generate Barcodes for Stock Locations",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "category": "Tools",
-    "author": "LasLabs, " "Odoo Community Association (OCA)",
-    "website": "https://laslabs.com",
+    "author": "LasLabs, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-barcode",
     "license": "AGPL-3",
-    "depends": ["barcodes_generator_abstract", "stock",],
-    "data": ["views/stock_location.xml",],
+    "depends": ["barcodes_generator_abstract", "stock", "stock_barcodes"],
```

Comment:
This dependency shouldn't be used. It's too much for this.

Reply:
@pedrobaeza  take a look at the changes I made regarding this, please.

Reply:
Would it be more appropiate to pull an extra `stock_location_barcode` module which both this and `stock_barcodes` depends on to keep this stuff simple?

Reply:
I don't think we should add such module complexity, and more not having control over enterprise module `stock_barcode`, so better to inject it via code if missing.